netZooPy/panda/panda.py

```diff
@@ -3,21 +3,21 @@
 import math
 import time
 import pandas as pd
-import numpy as np
 from scipy.stats import zscore
-from .timer import Timer
+# from .timer import Timer
```

Comment:
@dcolinmorgan why did you comment this line?

Reply:
oh damn

Reply:
ok i will delete this PR and try again :(